typedoc-multiple.json

```diff
@@ -0,0 +1,9 @@
+{
+  "entryPoints": [
+    "./dummy-projects/project1/index.ts",
+    "./dummy-projects/project2/index.ts"
```

Comment:
Would it work well like this?
"./dummy-projects/project1/aa/index.ts",
"./dummy-projects/project2/bb/index.ts",

Reply:
That's not possible at the moment, you should use `@module` tag within the `index.ts` file of each project instead.